testing/trino-tests/src/test/java/io/trino/security/TestAccessControl.java

```diff
@@ -292,11 +293,15 @@ private void reset()
         groupProvider.reset();
     }
 
-    @Test
-    public void testAccessControl()
+    @BeforeEach
```

Comment:
Please see https://trino.io/docs/current/develop/tests.html#keep-test-setup-and-teardown-simple